test/e2e/aio/cluster_test.go

```diff
@@ -113,6 +115,14 @@ func TestCluster(t *testing.T) {
 
 			configProxy := proxy.NewSecureClient("contrail", "config1-config-statefulset-0", 8082)
 
+			keystoneCRD := &contrail.Keystone{}
+			err := f.Client.Get(context.TODO(),
+				types.NamespacedName{
+					Namespace: namespace,
+					Name:      "keystone",
+				}, keystoneCRD)
+			assert.NoError(t, err)
```

Comment:
I guess you should require noerror here since without keystone CR it won't work

Reply:
changed to require